### PR TITLE
[JA] Fix typo Beatmapping/ja

### PR DESCRIPTION
**Pull request by lolita-is-godddd**
modify sentence, fix typo

---

### CODE REVIEW

**Comment by sr229** [comment]
Hey @lolita-is-godddd, next time when you make a one-commit PR, make sure you change the commit message. :wink:

**Comment by lolita-is-godddd** [comment]
ah sry and I will rell you
the first change, moved "または" (means "or") to between two things for fix japanese syntax.
second, deleted "も" ("is also") for syntax.
third, changed Rankable to "Ranked可能" ("can be ranked").
forth, changed "ビートスペーシング" to "ビートのすき間" for natural translation.
fifth, deleted unnessary "r", fixing typo.

File: wiki/Beatmapping/ja.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ A couple of rules of thumb are:-
 Beat Spacingってなんですか?
 ---------------------------
 
-<img src="Beatspacing.jpg" title="fig:A quick illustration" alt="A quick illustration" width="200" /> 直感的なプレイのために譜面はよい"ビートスペーシング"を持つべきです。一般的にタイムライン上で互いに近いBeatは、配置上でも近くなるべきで、逆も同様です。これは調和した譜面によって曲の流れを保証し、譜面をより楽しくさせることに繋がります。
+<img src="Beatspacing.jpg" title="fig:A quick illustration" alt="A quick illustration" width="200" /> 直感的なプレイのために譜面はよいビートのすき間を持つべきです。一般的にタイムライン上で互いに近いBeatは、配置上でも近くなるべきで、逆も同様です。これは調和した譜面によって曲の流れを保証し、譜面をより楽しくさせることに繋がります。
```
Comment (cl8n) [inline wiki/Beatmapping/ja.md]: すき間 -> 隙間 ?

Comment (lolita-is-godddd) [reply wiki/Beatmapping/ja.md]: fixed in `0a8f946`, check it out please?

**Pull request merged**